config/settings/base.py

```diff
@@ -161,7 +165,7 @@
         # https://docs.djangoproject.com/en/dev/ref/settings/#std:setting-TEMPLATES-BACKEND
         "BACKEND": "django.template.backends.django.DjangoTemplates",
         # https://docs.djangoproject.com/en/dev/ref/settings/#dirs
-        "DIRS": [str(APPS_DIR / "templates")],
```

Comment:
I'm not sure I follow the reason for this change.

Reply:
Aha, this is to include the templates for django-admin, as we need them to set flags!